inspire/dojson/common/base.py

```diff
@@ -39,6 +39,12 @@ def control_number(self, key, value):
     return value[0]
 
 
+@hep2marc.over('001', 'control_number')
+def control_number2marc(self, key, value):
+    """Record Identifier."""
+    return [value]
```

Comment:
@jirikuncar what do you mean? Anyway repeatable `controlfield` is really not valid MARC even in the case of liberal. (I mean, even without attaching any semantic to fields and subfields, a controlfield MUST be unique in any circumstances), so is perfectly fine to have that logic in `create_record`. Any way it's just a convention. As long as all layers agree on what to expects (currently lists even for controlfields).